modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,131 @@
+= Configure Role-based Access Control
+:description: Role-based access controls provide an alternative to ACLs for managing permissions.
+:page-categories: Management, Security
+
+Role-based Access Control (RBAC) provides a way to configure permissions for provisioned users at scale. RBAC allows you to manage access for many users to many resources with a streamlined interface. This mechanism works in conjunction with all supported authentication methods.
+
+IMPORTANT: At present, Redpanda only supports RBAC for self-hosted clusters. Configuration is only supported through the command line using RPK.
+
+== RBAC
+
+RBAC is designed to address the critical challenge of access management at scale. It alleviates the process of manually maintaining and verifying a set of raw permissions (ACLs) for a user base that may contain thousands of users. RBAC provides you with a method to onboard new employees easier, audit accesses faster, and adapt more readily to evolutions of usage and compliance needs.
+
+Using RBAC, you can define roles to reflect organizational structure or job duties to gives a coherent approach to access management. This approach decouples users and permissions through the assignment of roles. This allows a one to many mapping of a given role to many users, thereby dramatically reducing the number of custom policies needed for a resource from one per user to one per group of users.
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+=== RBAC terminology
+
+Under the RBAC framework, you create *roles*, grant *permissions* to those roles, and assign the roles to *users*. When you change the permissions for a given role, all principals with that role automatically gain the modified permissions. You grant or deny permissions for a role by creating an ACL and specifying the RBAC role as the allowed or denied principal respectively.
+
+==== Roles
+
+A role is a named collection of ACLs which can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
+
+* `user` - All authorized users have this role by default. There are no ACLs assigned to this role until you add them. This means users have no permissions to interact with any resources you add to your Redpanda cluster until you add relevant permissions to this or another role.
```

Comment:
`s/user/Users/` per PRD, though I'm not opinionated on naming.

Reply:
>authorized users have this role

Authorization status shouldn't come into play here. As I understand it, this should just apply to SASL/SCRAM users existing in the system at the time of cluster upgrade to an RBAC-supporting version of redpanda.

@deniscoady  - is this supposed to apply to every SASL user created, in perpetuity?

Reply:
I can update it to 'Users' for the role name. Is the other 'Admins' then? I was unclear as to whether that was the explicit role name or a plural reference to that type of account. Can we confirm that it was implemented pluralized as the PRD specified it @oleiman ?

Reply:
Confirmed it is "Users" in the PRD. "Admin" has been dropped and will not be included.

Also confirmed with Denis that we will assign this role ONLY to already extant SASL/SCRAM users _at the time of upgrade_. I still suspect that some finesse from product is required with respect to messaging.

Reply:
The doc should match whatever is in the code/PRD. Pick one and be consistent. This should be decided by @deniscoady.